comments/templatetags/comments.py

```diff
@@ -66,3 +66,7 @@ def get_latest_version(context):
     if context['node'].versions.count():
         return context['node'].versions.all()[0]
     return None
+
+@register.simple_tag(takes_context=True)
+def get_child_count(context):
+    return context['node'].get_children().filter(deleted=False).count()
```

Comment:
I think this is the wrong way to handle this. I think this is going to add one query for every comment with children. 
can we do something like 
`Comment.objects.filter(<filters>),annotate(child_count=Count(Comment.objects.filter(parent=OuterRef('pk'), deleted=False)))`